app/templates/main/accessibility.html

```diff
@@ -2,239 +2,177 @@
 {%- from 'govuk_frontend_jinja/components/back-link/macro.html' import govukBackLink -%}
 {%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
 {% block pageTitle %}Accessibility statement – {{ config['SERVICE_NAME'] }} – GOV.UK{% endblock %}
-{% block beforeContent %}
-    {{ super() }}
-    {{ govukBackLink({"text": "Back", "href": "main.index"}) }}
-{% endblock %}
 {% block content %}
-    <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
-            {{ super() }}
-            <h1 class="govuk-heading-xl">Accessibility statement for {{ config['SERVICE_NAME'] }}</h1>
-            {{ govukInsetText({
-            'text': "Note: start with a brief explanation of which websites or mobile apps the statement covers.
-            You can have a single accessibility statement that covers multiple domains, or a separate statement for each domain or subdomain. As long as the user can access relevant accessibility information easily from any page on your website."
-            }) }}
-            <p class="govuk-body">This accessibility statement applies to {{ config['SERVICE_URL'] }}.</p>
-            {{ govukInsetText({
-            "text": "Note: use this section to make a brief, general statement about what the website allows disabled users to do. Base it on the evaluation covered in detail in the ‘Technical information about this website’s accessibility’ section. If you’re not confident that something is accurate, leave it out. If you’re not confident enough to say anything specific here, leave this section out completely."
-            }) }}
-            <p class="govuk-body">
-                This website is run by {{ config['DEPARTMENT_NAME'] }}. We want as many people as possible to be able
-                to use this website. For example, that means you should be able to:
-            </p>
-            <ul class="govuk-list govuk-list--bullet">
-                <li>change colours, contrast levels and fonts</li>
-                <li>zoom in up to 300% without the text spilling off the screen</li>
-                <li>navigate most of the website using just a keyboard</li>
-                <li>navigate most of the website using speech recognition software</li>
-                <li>
-                    listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and
-                    VoiceOver)
-                </li>
-            </ul>
-            <p class="govuk-body">We’ve also made the website text as simple as possible to understand.</p>
-            <p class="govuk-body">
-                <a class="govuk-link" href="https://mcmw.abilitynet.org.uk/">AbilityNet</a> has advice on making your device easier to use if you have a disability.
+    <main class="govuk-main-wrapper govuk-main-wrapper--terms-of-use">
```

Comment:
I know we spoke yesterday, but can you confirm where this content from the accessibility page came from? Is this the actual content we will put in, or just a placeholder?